.gitignore

```diff
@@ -9,4 +9,5 @@ requirements.txt
 # IDEs
 .idea/
 .vscode/
-.DS_Store
\ No newline at end of file
+.DS_Store
+explain.md
```

Comment:
what is explain.md?

Reply:
I can remove it. just a file I have locally